workers/unity/Packages/com.improbable.gdk.core/Config/ReceptionistConfig.cs

```diff
@@ -9,6 +9,7 @@ public class ReceptionistConfig : ConnectionConfig
 
         // Default Android emulator host IP alias
         private const string AndroidEmulatorHostDeviceIp = "10.0.2.2";
+        private const string LocalHostIp = "127.0.0.1";
```

Comment:
we already store that ip in the RuntimeConfigDefaults so no need to redefine it :)